src/distributed/redis/config.ts

```diff
@@ -4,4 +4,5 @@ export interface RedisConfig {
   enabled: boolean
   connection: string | ClusterNode[]
   options: RedisOptions | ClusterOptions
+  scope: string
```

Comment:
While most of the codebase has no typings for the config.json file, here it happens that the redis service uses typings sur I added the scope property here.